### ISSUE TITLE
Usage/upkeep of land incentive

### ISSUE THREAD

**Issue by darikconley** [decentraland/proposals#127]
If someone is not taking care of a parcel that is located physically in an area which can likely produce income, that parcel should be moved to a specific set area of dcl while a new individual rents the parcels location for something active. The rent can be adjusted according to fair market rates but should not provide the absentee owner any incentive to abandon.

**Comment by Dmitry1987** [comment]
Good idea, idle parcels should "expire" and become available for purchase if not developed.  Otherwise lands that were bought by investors just to HODL them, will ruin game experience.

**Comment by collac** [comment]
how long does the land has to be empty to be claimed by someone else?

**Comment by Dmitry1987** [comment]
I think depends on proximity to "developed" lands. If it's surrounded by developed parcels - 1 month.
If it's remote - 1 year.

**Comment by daxadax** [comment]
How does that make sense?  If I want to leave my land empty, I'm free to do it - and what determines if a land is "idle"?  

I read one proposal that said land which hasn't been touched should begin to grow weeds, then trees, then become a small forest - I think I like that more than turning DCL into a homeowner's association.

**Comment by Dmitry1987** [comment]
good proposal @daxadax , i haven't seen this, can be a solution for empty lands, true.  it's just ugly to see miles and miles of land that investors bought and want to hold lets say for 5 years - it kills the purpose of the VR world - it must be full of content and fun to explore.

at the moment seeing the amounts of money invested in the auction - especially land near roads (1k$-10k$+ per parcel) and crazy bids near central plaza (I think I saw a lot of 40-50k$+ bids on single parcel)  -  it seems people will develop stuff for sure on their land. Otherwise how do they expect to make it back, those amounts of money.

**Comment by silkwebware** [comment]
Proposal - Management Contract (including fee payment) Allows employment of designers etc.
https://github.com/decentraland/proposals/issues/132
Not all speculators want to develop land. Make it easy for them to have the land developed by 3rd party. 
Incentives for designers/developers adding to the economy and avoiding stagnation. 

Also, we need a way to easily contact land land owners? I'm not in to enforcement and I believe in anonymity so if there is a way to do this we should discuss.

**Comment by Funkcikle** [comment]
I'm firmly opposed to this idea, barren patches of wasteland add character to any city, where will the kids hang out and get upto naughty things ?

Think of the children !!

**Comment by silkwebware** [comment]
They can hang out in between the Red Light & Fashion districts like in most cities. Haha.
